pkg/unshare/unshare.go

```diff
@@ -26,6 +26,7 @@ func HomeDir() (string, error) {
 				return
 			}
 			homeDir, homeDirErr = usr.HomeDir, nil
```

Comment:
Would it be better to change the variable homeDir to just home here?  That way it would work without the return and we could avoid someone accidentally removing the return later.